src/main/java/org/openjdk/jextract/impl/MacroParserImpl.java

```diff
@@ -337,7 +337,7 @@ void updateTable(TypeMaker typeMaker, Cursor decl) {
 
         void reparseMacros(boolean recovery) {
             String snippet = macroDecl(recovery);
-            TreeMaker treeMaker = new TreeMaker();
+            TreeMaker treeMaker = new TreeMaker(); // @@@: what about de-duplicated declarations?
```

Comment:
The comment is meant to indicate that, since we're creating a brand new TreeMaker, all the types we create in here, esp. those with a declared type, will point to declaration that are NOT deduplicated. I have a followup fix for this.